src/v/cloud_topics/reader/placeholder_extent.cc

```diff
@@ -122,80 +124,7 @@ model::record_batch placeholder_extent::make_raft_data_batch() {
     return batch;
 }
 
-ss::future<result<bool>> placeholder_extent::materialize(
```

Comment:
as far as i can tell this function was moved down in the the same file by about 100 lines, but i can't tell if it changed or not without examining the ~150 +/- lines 1 by 1. can you move it back to where it was at so the diff renders minimally?